pages/heroku.md

```diff
@@ -131,6 +131,115 @@ Or when using gradle:
 
 `heroku deploy:jar build/libs/*jar`
 
+## Deploying Docker to Heroku
+
+You can deploy your app as a Docker container to Heroku too. While this works, there's no Heroku setup and configuration that happens, so you have to do that manually. This documentation assumes you've already run `jhipster heroku` to deploy your app and therefore leverages the integration and add-on provisioning that this process performs.
+
+Build your Docker image:
+
+```
+./mvnw package -Pprod verify jib:dockerBuild
+```
+
+If you're using Gradle:
+
+```
+./gradlew -Pprod bootJar jibDockerBuild
+```
+
+You can test it out locally using Docker Compose.
+
+```shell
+docker-compose -f src/main/docker/app.yml up
+```
+
+Once you've confirmed everything works, create a new app on Heroku, and add it as a remote.
+
+```shell
+heroku apps:create
+git remote add docker https://git.heroku.com/<your-new-app>.git
+```
+
+Then run the commands below to deploy your JHipster app as a Docker image. Be sure to replace the `<...>` placeholders with your Heroku app name. If you don't know your app name, run `heroku apps`.
+
+```shell
+heroku container:login
+docker tag space registry.heroku.com/<heroku-app>/web
+docker push registry.heroku.com/<heroku-app>/web
+```
+
+For example:
+
+```shell
+heroku container:login
+docker tag space registry.heroku.com/fast-peak-70014/web
+docker push registry.heroku.com/fast-peak-70014/web
+```
+
+At this point, you can use the PostgreSQL and Okta add-ons you've already configured. Run the following command to get the identifiers of the add-ons from the `heroku` remote that you first deployed to.
+
+```shell
+heroku addons --remote heroku
+```
+
+Then you can attach these instances to your new application.
+
+```shell
+heroku addons:attach <postgresql-addon-name> --remote docker
+heroku addons:attach <okta-addon-name> --remote docker
+```
+
+When you use `jhipster heroku` to deploy your application, it properly configures the database for you. However, when deploying it as a Docker container, none of that happens. Therefore, you need to set a few configuration variables so your Docker container can talk to PostgreSQL. First, run the following command to get the PostgreSQL URL.
+
+```
+heroku config:get DATABASE_URL --remote docker
+```
+
+Then, set the database environment variables to match the keys that are in `application-heroku.yml`:
+
+```shell
+heroku config:set JDBC_DATABASE_URL=<value-after-@-from-last-command> --remote docker
+heroku config:set JDBC_DATABASE_USERNAME=<username-value-from-last-command> --remote docker
+heroku config:set JDBC_DATABASE_PASSWORD=<password-value-from-last-command> --remote docker
+```
+
+Set the max amount of Java memory to use and specify the Spring profiles.
+
+```shell
+heroku config:set JAVA_OPTS=-Xmx256m
+heroku config:set SPRING_PROFILES_ACTIVE=prod,heroku,no-liquibase
```

Comment:
I am not sure you should set `no-liquibase` here. If there are new changes and you don't deploy it via git/jar before liquibase won't run with the docker container (as it is run during release hook via dedicated mvn command). For gradle we need the liquibase part in any case I think.

Reply:
OK, I'll remove it.